articles/200_migration_guide_from_ros1.md

```diff
@@ -162,3 +162,483 @@ NOTE: to be written
 ## Launch files
 
 While launch files in ROS 1 are specified using [.xml](http://wiki.ros.org/roslaunch/XML) files ROS 2 uses Python scripts to enable more flexibility (see [launch package](https://github.com/ros2/launch/tree/master/launch)).
+
+## Example: Converting an existing ROS 1 package to use ROS 2
+
+Let's say that we have simple ROS 1 package called `talker` that uses `roscpp`
+in one node, called `talker`.  This package is in a catkin workspace, located
```

Comment:
done

While I understand the reason for doing this (more concise diffing), I find that style incredibly tedious and a poor tradeoff (because it makes the experience of reading, writing, or editing the underlying text in an editor much worse).